packages/itwinui-react/src/core/Table/filters/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -117,25 +118,33 @@ export const DateRangeFilter = <T extends Record<string, unknown>>(
 
   return (
     <BaseFilter>
-      <DatePickerInput
-        label={translatedStrings.from}
-        date={from}
-        onChange={onFromChange}
-        formatDate={formatDate}
-        parseInput={parseInput}
-        onKeyDown={onKeyDown}
-        placeholder={placeholder}
-        setFocus
-      />
-      <DatePickerInput
-        label={translatedStrings.to}
-        date={to}
-        onChange={onToChange}
-        formatDate={formatDate}
-        parseInput={parseInput}
-        onKeyDown={onKeyDown}
-        placeholder={placeholder}
-      />
+      <DateRangeFilterContext.Provider
+        value={{ datePicker: 'from', selectedDate: to }}
```

Comment:
why not just pass these as props instead of context?

Reply:
also unit tests are failing because of useSafeContext

Reply:
This has been fixed